CMakeLists.txt

```diff
@@ -335,6 +335,20 @@ else()
     endif()
 endif()
 
+if (ONEDPL_BACKEND MATCHES "^(pstl_offload)$")
+    message(STATUS "Adding pstl_offload compilation flags")
+    # TODO: only special compiler option should be added after implementing pstl_offload support in the compiler
+    # target_compile_options(oneDPL INTERFACE "-fsycl-pstl-offload")
+    target_include_directories(oneDPL INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/include/pstl_offload)
+    target_compile_options(oneDPL INTERFACE "-D__SYCL_PSTL_OFFLOAD__=1")
```

Comment:
In the final version adding `pstl_offload` to include path, setting the macro and linkage with `libpstloffload` would be done on the compiler side by setting `-fsycl-pstl-offload`. 
I did not remove it because for today we don't have the release compiler that supports this option and we use this "manual" options substitution.